docs/settings/alert-action-settings.asciidoc

```diff
@@ -0,0 +1,47 @@
+[role="xpack"]
+[[alert-action-settings-kb]]
+=== Alerting and action settings in Kibana
+++++
+<titleabbrev>Alerting and action settings</titleabbrev>
+++++
+
+Alerts and actions are enabled by default in {kib}, but require you configure the following in order to use them: 
```

Comment:
This does not feel like the right place for this  setup information, but since alerting and actions are not yet exposed in the UI I'm not sure it warrants it's own configuration section yet.